util/exec_test.go

```diff
@@ -91,3 +92,35 @@ func TestFilterArgs(t *testing.T) {
 		},
 	})
 }
+
+func TestParallelExecute(t *testing.T) {
+	type testCase struct {
+		Limit uint
+		Times uint
+
+		Function func()
+	}
```

Comment:
Kinda weird having table tests that are not used in any table fashion. Would drop them.

Reply:
Still weird cause we construct them within the tests and then just use their properties. So would either introduce a more elaborate `validate` or really just remove the `testCase`. To me it makes everything harder to read.